docs/hardware/devices/linux-native-hardware/linux-native-hardware.mdx

```diff
@@ -114,13 +136,120 @@ dtoverlay=uart0
 
 - The correct port for UART GPS on the Pi 5 after a reboot is `/dev/ttyAMA0`.
 - The correct port for UART GPS on earlier Pi versions after a reboot is `/dev/ttyS0`.
+- You may need to disable the serial console on a Pi and then reboot
+```shell
+sudo raspi-config nonint do_serial_cons 1 # Disable Serial Console
+```
 
 ### Persistence
 
 - The persistent .proto db files of the portduino version of meshtasticd are stored under: `/root/.portduino/default/prefs/`.
 
 ### Advanced Setup and Troubleshooting
 
+#### Creating and Enabling a Systemctl Script (non .deb installs)
+
+To Setup the device to start and stop meshtasticd as as service using systemctl you can setup the service unit using the instructions below.
+
+Create the service unit file:
+
+Create a new file in the /etc/systemd/system/ directory with a name like meshtasticd.service.
+```shell
+sudo nano /etc/systemd/system/meshtasticd.service
+```
+Add the following content to the file:
+
+```plaintext
+[Unit]
+Description=Meshtastic Daemon
+After=network.target
+
+[Service]
+ExecStart=/usr/sbin/meshtasticd
+Restart=always
+User=root
+Group=root
+StandardOutput=file:/var/log/meshtasticd.log
+StandardError=file:/var/log/meshtasticd.log
+Type=simple
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Reload systemd to recognize the new service:
+```shell
+sudo systemctl daemon-reload
+```
+
+Enable the service to start on boot:
+```shell
+sudo systemctl enable meshtasticd
+```
+
+##### Starting and Stopping the Service
+
+Start the service:
+```shell
+sudo systemctl start meshtasticd
+```
+
+Check the status of the service:
+```shell
+sudo systemctl status meshtasticd
+```
+
+Stop the service:
+```shell
+sudo systemctl stop meshtasticd
+```
+
+This will give you a detailed view of the service status and any potential errors.
+
+By following these steps, you set up a systemd service for meshtasticd that will start automatically at boot and restart if it crashes. You can manage it using the standard systemctl commands (start, stop, restart, status, etc.).
+
+#### Setup Log Rotation
```

Comment:
And so here, we could simply inform users that `journalctl -u meshtasticd -b` will fetch the meshtasticd logs since boot, and `journalctl -u meshtasticd -f` will follow the live log output.

Reply:
Makes sense, removed Logrotate script and updated with information on how to view logs from the journal.